music21/articulations.py

```diff
@@ -535,7 +540,12 @@ class NailPizzicato(Pizzicato):
     pass
 
 class FretIndication(TechnicalIndication):
-    pass
+    def __init__(self):
```

Comment:
As above on StringIndication.  number should default to 0.  So None should not be allowed